realm/src/main/java/io/realm/RealmQuery.java

```diff
@@ -169,6 +169,30 @@ private boolean containsDot(String s) {
         }
     }
 
+    // isNull and isNotNull
```

Comment:
Having these kind of sections is not really maintainable. So I would prefer if we got rid of them.